src/nbtypes.jl

```diff
@@ -99,3 +101,68 @@ function Base.show(io::IO, m::KernelNB)
     println(io, "  Classes = $(keys(m.c_kdes))")
     print(io, "  Number of variables = $(m.n_vars)")
 end
+
+#####################################
+#####  Hybrid Naive Bayes       #####
+#####################################
+""" a wrapper around key value pairs for a discrete probability distribution """
+immutable ePDF{C <: Associative}
```

Comment:
Hi guys, I'm a bit late to the party. Wouldn't this be much simpler with Multinomial for discrete, kernel for continuous and then combine the log odds  for hybrid (since we are being naive)? I guess it depends on what you mean by discrete as well, is it "dogs" and "cats" or is it "counts". I think using the kde make a lot of sense of the latter...